returnn/hdf.py

```diff
@@ -195,3 +199,80 @@ def create_files(self):
             f.write(
                 '<?xml version="1.0" ?>\n<network><out name="features" /></network>'
             )
+
+
+class BlissToPcmHDFJob(Job):
+    """
+    Gets audio files from a Bliss corpus and stores them as HDF file
+    compatible with the returnn HDFDataset
+    """
+
+    def __init__(
+        self,
+        bliss_corpus: tk.Path,
+        segment_file: Optional[tk.Path] = None,
+        output_dtype: str = "int16",
+        returnn_root: Optional[tk.Path] = None,
+    ):
+        """
+
+        :param bliss_corpus: Bliss corpus to read segments and audio files from
+        :param segment_file: segment file that lists allowed segments
+        :param output_dtype: dtype that should be written in the hdf (supports float64, float32, int32, int16)
+        :param returnn_root: returnn repository
+        """
+        self.set_vis_name("Dump audio to HDF")
+
+        self.bliss_corpus = bliss_corpus
+        self.segment_file = segment_file
+        assert output_dtype in ["float64", "float32", "int32", "int16"]
+        self.output_dtype = output_dtype
+        self.returnn_root = returnn_root
+
+        self.out_hdf = self.output_path("audio.hdf")
+
+    def tasks(self):
+        yield Task("run", mini_task=True)
```

Comment:
Are you sure this is always suitable as a mini task even for large corpora?

Reply:
I am always dumping HDFs with ~10h of audio. This takes ~5 min.

If one were to dump the whole of LibriSpeech into one file, then one certainly would need a cluster node.

Maybe we should make it always a full task, even for the 5 min case.

Reply:
I am not sure where, but I remember seeing this in some jobs as an internal flag, where the size of the corpus is checked and depending on size mini_task is either set or not. But I cant recall where right now.

Reply:
This would require resolving a Path object (the corpus) in the manager. I do not want to do that.

Reply:
Yes, then making it a full task seems reasonable to me.

Reply:
So I found this: 
https://github.com/rwth-i6/i6_core/blob/e904b981a8fce46caaf449f252a2c4f2a00e44ad/datasets/huggingface.py#L37
there it is set by the user manually. I am not sure if this makes sense here, otherwise full task makes sense to me.
Most likely what I saw was dependent on some user set input parameter (like size or so), but I am not 100% sure anymore.

Reply:
The `CorpusObject` from `meta/system.py` has an attribute `duration`, this could also be related. But here, the input is a `tk.Path`.